br/use-br-command-line-tool.md

```diff
@@ -244,6 +244,16 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 
 In the above example, for the incremental backup data, BR records the data changes and the DDL operations during `(LAST_BACKUP_TS, current PD timestamp]`. When restoring data, BR first restores DDL operations and then the data.
 
+### Point in Time recovery (experimental feature)
+
```

Comment:
@kennytm Could you help explain what this feature is, the user scenarios, and what users should pay attention to?

Reply:
The senario:

- Backup is taken every 3 hours (06:00, 09:00, 12:00, ...)
- TiCDC is enabled

Now when at 08:00 someone accidentally truncates the wrong table:
- The backup from 06:00 gets restored
- Then after this restore the `cdclogs` from 06:00 until just before 08:00 are applied

The result is that the RPO (Restore Point Objective, time in history to which the database gets restored) can be chosen freely. Without this the changes between 06:00 and 08:00 whould have been lost.

Reply:
Could you give a short summary of what Point in Time recovery is (like "Point in Time recovery is an experimental feature introduced in vx.x that allows you to freely choose the RPO... For example, suppose that BR performs backup every three hours and TiCDC is enabled...") and then the example?